src/patternfly/layouts/Split/examples/Split.md

```diff
@@ -87,13 +87,13 @@ import './Split.css'
 
 ## Documentation
 ### Overview
-The split layout is designed to position items horizontally, with one item filling the available horizontal space.
+The split layout is designed to position items horizontally.
```

Comment:
AFAIK there is no expectation that any (or just one) item(s) need to fill the space.